packages/plugin-repo-cli/commands/analyzePlugin.ts

```
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+
+import { execCommand } from '@joplin/utils';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
+export default async function(args: any) {
+    const pluginId = args.pluginId;
```

ESLint

[error] Expected indentation of 1 tab but found 4 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 4 spaces.

(indent)